### PR TITLE
Add necessary tip to avoid misunderstanding

### PR DESCRIPTION
**Pull request by unbyte**
### What is changed, added or deleted? (Required)

Add "local access only" for `tiup playground`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 
 ## 第一种：使用 TiUP Playground 快速部署本地测试环境
 
-- 适用场景：利用本地 Mac 或者单机 Linux 环境快速部署 TiDB 集群。可以体验 TiDB 集群的基本架构，以及 TiDB、TiKV、PD、监控等基础组件的运行。
+- 适用场景：利用本地 Mac 或者单机 Linux 环境快速部署仅本地可访问的 TiDB 集群。可以体验 TiDB 集群的基本架构，以及 TiDB、TiKV、PD、监控等基础组件的运行。
```
Comment (lonng) [inline quick-start-with-tidb.md:20]: 默认是监听 `127.0.0.1` 只能本地访问，可以通过 `--host=0.0.0.0` 或者其他网卡绑定可其他网络访问的 IP 时，可以由其他机器访问。

Comment (unbyte) [reply quick-start-with-tidb.md:20]: @lonng 使用`0.0.0.0`时，tikv将报错 https://github.com/tikv/tikv/blob/3fcbdc350a734e963fb15122d53695fbf667080e/src/server/config.rs#L196-L201

Comment (unbyte) [reply quick-start-with-tidb.md:20]: 在`tiflash_tikv.log`中的输出
```
[2020/09/29 20:53:05.955 +08:00] [FATAL] [setup.rs:212] ["invalid configuration: [src/server/config.rs:198]: invalid advertise-addr: \"0.0.0.0:20170\""]
```
等待timeout之后验证确实能够外部访问

**Review by lonng — COMMENTED** [review]
LGTM

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -92,6 +92,10 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
     tiup clean --all
     ```
 
+> **注意：**
+>
+> Playground 默认监听 127.0.0.1, 服务仅本地可访问；若需要使服务可被外部访问，您可以通过 `--host` 参数指定监听 `0.0.0.0` 或网卡绑定的可外部访问的 IP
```
Comment (ran-huang) [inline quick-start-with-tidb.md:97]:
```suggestion
> TiUP Playground 默认监听 127.0.0.1，服务仅本地可访问；若需要使服务可被外部访问，您可以通过 `--host` 参数指定监听 `0.0.0.0` 或网卡绑定的可外部访问的 IP。
```

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -92,6 +92,10 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
     tiup clean --all
     ```
 
+> **注意：**
+>
+> TiUP Playground 默认监听 127.0.0.1，服务仅本地可访问；若需要使服务可被外部访问，您可以通过 `--host` 参数指定监听 `0.0.0.0` 或网卡绑定的可外部访问的 IP。
```
Comment (TomShawn) [inline quick-start-with-tidb.md:97]:
```suggestion
> TiUP Playground 默认监听 `127.0.0.1`，服务仅本地可访问；若需要使服务可被外部访问，你可以通过 `--host` 参数指定监听 `0.0.0.0` 或网卡绑定外部可访问的 IP。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**